beetsplug/export.py

```diff
@@ -52,6 +54,23 @@ def __init__(self):
                     'sort_keys': True
                 }
             },
+            'csv': {
+                # csv module formatting options
+                'formatting': {
+                    'delimiter': ',', # column seperator
+                    'dialect': 'excel', # the name of the dialect to use
```

Comment:
This option seems useful enough! But maybe it's worth linking (in the documentation) to the place in the Python docs where dialects are described?

Reply:
Oh yes I forgot to do that, thank you for reminding me